staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder.go

```diff
@@ -308,6 +308,8 @@ func (b *builder) buildKubeNative(schema *structuralschema.Structural, v2 bool)
 			schema = ToStructuralOpenAPIV2(schema)
 		}
 		ret = schema.ToGoOpenAPI()
+		// TODO(roycaihw, 1.16): go through unfolding and v3->v2 conversion following the KEP, if the root schema
```

Comment:
the root cannot be x-kubernetes-embedded-resource. Compare https://github.com/kubernetes/kubernetes/blob/bada1c6b1eef959825c3dca1d3944e1ac4c31184/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go#L863